gramps/gui/views/treemodels/placemodel.py

```diff
@@ -124,41 +123,39 @@ def on_get_n_columns(self):
         return len(self.fmap) + 1
 
     def column_title(self, data):
-        handle = data["handle"]
```

Comment:
Have you benchmarked `data["handle"]` vs `data.handle`?
For `data["handle"]` is the performance of the DataDict version the same as builtin dict?
Instinctively, I'd expect `data.handle` to be slightly slower. DataDict.__get_attr__ is doing extra work in the normal case:

1. a string comparison `key.startswith("_")`
2. lookup in dict: `key in self`
and then repeat for `self[key]`. Shame we cannot optimise this to avoid the double lookup
3. two x calls to `isinstance()`


I did see your comment in PR #1824 where you reported that `person.get_gender()` was 4.4 times slower than `person.gender`

Reply:
I'll do a test, but the cost should be negligible. The get_gender() is 4 times slower because it instantiates an object.  The gain in readability and flexibility should be also weighed.

Reply:
Great, thanks for doing the benchmark.
I'm strongly in favour of the change so apologies if my original comment came across negatively. It was not intended!

Reply:
No, all good points!

Reply:
I think this one can be resolved.

Reply:
Had to instantiate the full object in order to work with proxies.